provider/cldr/src/lib.rs

```diff
@@ -19,6 +19,16 @@
 //! [data provider]: icu_provider
 //! [`FsDataProvider`]: ../icu_provider_fs/struct.FsDataProvider.html
 
+#![cfg_attr(
+    not(test),
+    deny(
```

Comment:
Request: For now, exclude this particular crate (`icu_provider_cldr`) from the change. This crate is being tracked separately in #574

Reply:
Request: Please also undo the rest of the changes to the files in this crate.

Reply:
Actually, never mind. What you did is fine.